src/Compilers/CSharp/Portable/Symbols/ErrorTypeSymbol.cs

```diff
@@ -645,7 +645,7 @@ internal sealed class SubstitutedNestedErrorTypeSymbol : SubstitutedErrorTypeSym
             base(originalDefinition)
         {
             _containingSymbol = containingSymbol;
-            _map = containingSymbol.TypeSubstitution.WithAlphaRename(originalDefinition, this, nonNullTypesContext: this, out _typeParameters);
+            _map = containingSymbol.TypeSubstitution.WithAlphaRename(originalDefinition, this, nonNullTypesContext: originalDefinition, out _typeParameters);
```

Comment:
>nonNullTypesContext: originalDefinition [](start = 95, length = 39)

Does `WithAlphaRename` depend on the actual `nonNullTypesContext` argument? If not, we should remove that argument and use a placeholder such as `NonNullTypesUnusedContext.Instance` internally in `WithAlphaRename`.